.pre-commit-config.yaml

```diff
@@ -0,0 +1,35 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+      - id: check-merge-conflict
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+
+-   repo: https://github.com/pre-commit/mirrors-yapf
+    rev: v0.31.0
+    hooks:
+    -   id: yapf
+
+-   repo: https://gitlab.com/PyCQA/flake8
+    rev: 3.8.4
+    hooks:
+    -   id: flake8
+        args: [--count]
+
+-   repo: local
+    hooks:
+      - id: pytest-check
+        name: pytest-check
+        types: [python]
+        entry: pytest -v -m 'slow'
+        language: system
+        pass_filenames: false
+        always_run: true
+
+-   repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
mypy tests take about 45 secs to complete. I feel that waiting for 45 sec for getting a commit done is not a efficient process.

Reply:
What do you suggest then ?